addons/common/functions/fnc_checkVersionNumber.sqf

```diff
@@ -21,20 +21,26 @@ if (canSuspend) exitWith {
 };
 
 params [["_whitelist", missionNamespace getVariable ["ACE_Version_Whitelist", []]]];
-
-private _files = CBA_common_addons select {
-    (_x select [0, 3] != "a3_") &&
-    {_x select [0, 4] != "ace_"} &&
-    {!((toLowerANSI _x) in _whitelist)}
-};
+TRACE_1("",_whitelist);
 
 private _cfgPatches = configFile >> "CfgPatches";
+private _files = [];
 private _versions = [];
 
 {
+    // Skip A3, ACE, and whitelisted addons
```

Comment:
Why would we want to skip the version check? This is a bad idea.

Reply:
We have never checked the versions of A3 and whitelisted addons. ACE has its own checks.

Reply:
Oh, I missed that. My bad.

Reply:
TBF, would benefit mentioning that in the comment:
```suggestion
    // Skip A3, ACE, and whitelisted addons, because we have already done the check.
```